base/reflection.jl

```diff
@@ -493,8 +493,8 @@ end
     ismutable(v) -> Bool
 
 Return `true` if and only if value `v` is mutable.  See [Mutable Composite Types](@ref)
-for a discussion of immutability. Note that this function works on values, so if you give it
-a type, it will tell you that a value of `DataType` is mutable.
+for a discussion of immutability. Note that this function works on values, so if you
+give it a `DataType`, it will tell you that a value of the `DataType` is mutable.
```

Comment:
```suggestion
give it a `DataType`, it will tell you that a value of the type is mutable.
```

Seems a bit excessive to mention `DataType` twice. After it has been used first, we know what type we talk about.